datanode/sqlstore/migrations/0049_maker_fees_in_fees_stats.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+
+alter table fees_stats
+    add column total_maker_fees_received jsonb not null default '[]';
+
+alter table fees_stats
+    add column maker_fees_generated jsonb not null default '[]';
+
+
+-- +goose Down
+
+alter table fees_stats
+    drop column total_maker_fees_received;
+
+alter table fees_stats
+    drop column maker_fees_generated;
+
```

Comment:
I guess this could have been added to the original migrations seing that there's no in between release, but doesn't matter so much

Reply:
As I didn't wanted to block any release I thought another migration would be wiser.